### ISSUE TITLE
Documentation for the conformance test suite

### ISSUE THREAD

**Issue by shaneutt** [kubernetes-sigs/gateway-api#2322]

**Comment by shaneutt** [comment]
This is a requirement for standard graduation of the KEP, and will be worked on soon, not stale.

/remove-lifecycle stale

**Comment by shaneutt** [comment]
Was talking with @mlavacca and he's gonna work on this last little bit!

/assign @mlavacca 

Thanks Mattia, we don't need anything huge, just a `conformance/README.md` which includes a section that describes how to use the conformance test suite for new implementations.

**Comment by shaneutt** [comment]
Also, now that this is the last thing we need to do in order to consider #1709 `Standard`, note that the PR which resolves this can also have the honors of updating the GEP to `Standard`! :partying_face:

**Comment by mlavacca** [comment]
> Also, now that this is the last thing we need to do in order to consider #1709 `Standard`, note that the PR which resolves this can also have the honors of updating the GEP to `Standard`! 🥳

🎉 

There is also another aspect I'm thinking about: we should remove the "experimental" prefix from the suite. And maybe add the "legacy" prefix to the current standard one?

**Comment by mlavacca** [comment]
> Good call, I think:
> 
> > we should remove the "experimental" prefix from the suite
> 
> Yes 👍
> 
> > and maybe add the "legacy" prefix to the current standard one?
> 
> The original intention was that the experimental one would simply merge in with the old one and become the old one, if that's feasible I think we should just go for that.

yes, I think it makes sense. I'll address the final migration in a different PR 👍
